packages/web/components/swap-tool/index.tsx

```
@@ -277,676 +320,306 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
+                              minimumSignificantDigits: 6,
+                              maximumSignificantDigits: 6,
+                              maxDecimals: 10,
+                              notation: "standard",
+                            }
+                          : undefined
+                      )
+                    }
+                    showAddFundsButton={
+                      !account?.isWalletConnected ||
+                      (swapState.inAmountInput.balance &&
+                        swapState.inAmountInput.balance.toDec().isZero())
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)